crates/api_models/src/payments.rs

```diff
@@ -5324,6 +5369,8 @@ pub struct OrderDetails {
     /// The quantity of the product to be purchased
     #[schema(example = 1)]
     pub quantity: u16,
+    pub tax_rate: Option<i64>,
```

Comment:
can you please delete this struct, I see that this is not being used anywhere.

Reply:
Are you asking to delete the tax_rate field or whole struct?

Reply:
The whole `OrderDetails` struct, if it is not being used anywhere

Reply:
It is being used, and even the struct was already created.